src/gallium/drivers/lima/lima_bo.c

```diff
@@ -180,8 +180,10 @@ int lima_bo_export(lima_bo_handle bo, enum lima_bo_handle_type type,
       *handle = bo->handle;
       return 0;
    case lima_bo_handle_type_dma_buf_fd:
-      /* unsupported yet */
-      return -EINVAL;
+      pthread_mutex_lock(&bo->dev->bo_table_mutex);
```

Comment:
Although amdgpu just write like this, I'd prefer we can do drmPrimeHandleToFD pre hash table set for error case correction.